sopel/modules/bugzilla.py

```diff
@@ -11,7 +11,7 @@
 import re
 
 import requests
-import xmltodict
+import xmltodict  # type: ignore
```

Comment:
It's necessary because `xmltodict` doesn't have type-hint and it doesn't have a stub. So mypy can't check type with it. So I have to ignore it, for *type* only. There is no other reason to add `type: ignore`, this is specifically for mypy and I don't know another way to do that.

Reply:
> It's necessary because `xmltodict` doesn't have type-hint and it doesn't have a stub.

Ah, of course. _\*mentally copy-pastes this explanation to all the other plugins where this is done\*_

> and I don't know another way to do that.

[mypy docs about error codes](https://mypy.readthedocs.io/en/stable/error_codes.html) say you can do `# type: ignore[error-code]`, and a couple other pages list the possible errors. The only thing is that mypy doesn't show error codes by default, so it would probably be useful (regardless of whether these annotations change to be more specific) to add `show_error_codes = True` to mypy's config.

Reply:
Yup, it was just `[import]`.